Flagsmith.FlagsmithClient/Flag.cs

```diff
@@ -0,0 +1,52 @@
+using Newtonsoft.Json;
+using FlagsmithEngine.Feature.Models;
+using FlagsmithEngine.Identity.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Flagsmith
+{
+    [JsonObject(MemberSerialization.OptIn)]
+    public class Flag
+    {
+        public Flag() { }
+        public Flag(Feature feature, bool enabled, string value)
+        {
+            this.Feature = feature;
+            this.Enabled = enabled;
+            this.Value = value;
+        }
+        [JsonProperty("featureId")]
+        protected int FeatureId;
+        [JsonProperty("feature")]
+        protected Feature Feature = null;
```

Comment:
As per above comment, this should just be `protected string FeatureName;` I think?

Reply:
should i replace it with Feature object ? because if we update it then JSON response and our Class schema will be different.